demisto_sdk/__main__.py

```diff
@@ -69,6 +70,22 @@ def __init__(self):
 pass_config = click.make_pass_decorator(DemistoSDK, ensure=True)
 
 
+def check_configuration_file(command, args):
+    config_file_path = '.demisto-sdk-conf'
+    if os.path.isfile(config_file_path):
+        try:
+            config = ConfigParser(allow_no_value=True)
+            config.read(config_file_path)
+
+            if command in config.sections():
+                for key in config[command]:
+                    if key in args:
+                        args[key] = config[command][key]
```

Comment:
I think it is better to have it the other way around:
the command arguments should override the default configuration